packages/aggregator/src/analytics/aggregateAnalytics/aggregations/getFinalValuePerPeriod.js

```diff
@@ -106,12 +106,18 @@ class FinalValueAggregator {
     this.cache = cache;
   }
 
-  getContinuousValues(analytics, aggregationPeriod) {
-    const periods = getContinuousPeriodsForAnalytics(analytics, aggregationPeriod, true);
+  getFilledValues(aggregationPeriod, fillEmptyPeriodsWith) {
+    const checkIfFillingContinuousValues = fillEmptyPeriodsWith === 'previous';
```

Comment:
```suggestion
    const fillingPreviousValues = fillEmptyPeriodsWith === 'previous';
```
Minor suggestion but reads a bit more clearly to me